src/Storages/ColumnsDescription.cpp

```diff
@@ -207,6 +207,9 @@ void ColumnDescription::readText(ReadBuffer & buf)
 
             if (col_ast->settings)
                 settings = col_ast->settings->as<ASTSetQuery &>().changes;
+
+            if (col_ast->statistics_desc)
```

Comment:
I don't really get the followings, probably because of my limited knowledge:
- why this change makes statistics work with RMT?
- why the call to `ColumnStatisticsDescription::fromColumnDeclaration` in `InterpreterCreateQuery` isn't enough to make statistics work with RMT?